BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -12,18 +12,24 @@ public struct Label: Element {
     public var alignment: NSTextAlignment = .left
     public var numberOfLines: Int = 0
     public var lineBreakMode: NSLineBreakMode = .byWordWrapping
-    /// The scale to which pixel measurements will be rounded. Defaults to `UIScreen.main.scale`.
-    public var roundingScale: CGFloat = UIScreen.main.scale
+    public var lineHeight: LineHeight = .font
+    public var isAccessibilityElement = false
 
     public init(text: String, configure: (inout Label) -> Void = { _ in }) {
         self.text = text
         configure(&self)
     }
     
     private var attributedText: NSAttributedString {
-        let paragraphStyle = NSParagraphStyle.default.mutableCopy() as! NSMutableParagraphStyle
+        let paragraphStyle = NSMutableParagraphStyle()
         paragraphStyle.alignment = alignment
         paragraphStyle.lineBreakMode = lineBreakMode
+
+        if case .custom(let lineHeight, _) = lineHeight {
+            paragraphStyle.minimumLineHeight = lineHeight
```

Comment:
Assume this defaults to a sane value without setting these defaults, right?

Reply:
Yeah, if it's not set it just uses whatever the font has built in

Reply:
(I'd still ask for a `switch`, in case there's more cases added)

Reply:
changed to switch